api/roleAccess/exclusionService.ts

```diff
@@ -71,6 +70,7 @@ export function prepareExclusionBody(currentUserId: string, assigneeId: string,
         caseId: body.caseId,
         jurisdiction: body.jurisdiction,
         notes: body.exclusionDescription,
+        caseType: body.caseType,
```

Comment:
Destructuring and property value shorthand could be used here to avoid repetition and simplify code e.g:

`const { caseId, caseType, exclusionDescription, jurisdiction } = body;`

...

```
{
  caseId,
  jurisdiction,
  notes: exclusionDescription,
  caseType,
}
```

Reply:
Yes but it would end up something like this
`      attributes: {
        caseId,
        jurisdiction,
        notes: body.exclusionDescription,
        caseType,
      },`

Reply:
Agree that the idea is right if all the property names match